guides/common/modules/proc_copying-a-content-view.adoc

```diff
@@ -0,0 +1,29 @@
+[id="Copying_a_Content_View_{context}"]
+= Copying a content view
+
+Use this procedure to copy a content view.
+You can copy a content view in the {ProjectWebUI} or you can use the Hammer CLI to copy an existing content view into a new content view.
+
+[NOTE]
+====
+A copied content view does not have the same history as a regular content view.
+Version 1 of the content view is where a copied content view begins.
+As a result, you cannot promote an older version of a content view from the copied content view.
+====
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Lifecycle* > *Content Views*.
+. Select the content view you want to copy.
+. Click the vertical ellipsis icon and click *Copy*.
+. In the *Name* field, enter a name for the content view you want to copy and click *Copy content view*.
+
+The copied content view will appear on the *Content views* page.
```

Comment:
```suggestion
.Verification
* The copied content view appears on the *Content views* page.
```
And verification goes below the _CLI procedure_.

Reply:
Yeah maybe it should. Both the Web UI and CLI procedures lead to that outcome.